package.json

```diff
@@ -15,9 +15,9 @@
     "typecheck": "tsc"
   },
   "dependencies": {
-    "@docusaurus/core": "^2.3.1",
+    "@docusaurus/core": "^2.4.1",
```

Comment:
Please take a look at if the dependencies changes are okay. I am not a newbie with `nodejs`. Please suggest changes if required. Thanks.

Reply:
Let's not update the versions in document change branches.